Hymns/Favorites/FavoritesView.swift

```diff
@@ -14,7 +14,10 @@ struct FavoritesView: View {
         VStack {
             CustomTitle(title: "Favorites")
             if self.viewModel.favorites.isEmpty {
-                Text("Tap the ♡ on any hymn to add as a favorite").maxSize()
+                ZStack {
+                    Image("favorites illustration").offset(y: -70)
+                    Text("Tap the ♡ on any hymn to add as a favorite").maxSize().offset(y: 70)
```

Comment:
How come you didn't just use a VStack?

Reply:
VStack was tough to get aligned in the right position. The Image is always way to high and text way to low. I could force them into only taking up so much space but thought that might not be too good for various devices. This ZStack route gave me what I want on various device sizes every time even when testing on iPad so it seems like as good of a way as any to me.

Reply:
what about

```                VStack {
                    Image("favorites illustration")
                    Text("Tap the ♡ on any hymn to add as a favorite")
                }.maxSize()```

Reply:
That works pretty well actually. I went ahead and committed it. At first it was too low for the spec but once an offset is added it looks good.